### PR TITLE
Add params.quality_gate for get step.

### PR DESCRIPTION
**Pull request by SimonXming**
`quality_gate` is for breaks the build if the analysis result is `ERROR` or `WARN`.

### CODE REVIEW

File: assets/common.sh [hunk 1]
```diff
@@ -91,3 +91,72 @@ function sq_server_version {
...
+    local qg_settings="$1"
+    local qg_status_path="$2"
+
+    echo "Start parseing quality_gates..."
+    local project_status=$(jq -r '.projectStatus.status // ""' < "$qg_status_path")
+    if [ "$project_status" == "OK" ]; then
+        return 0
+    fi
+
+    if [ "$project_status" == "WARN" ]; then
```
Comment (headcr4sh) [inline assets/common.sh]: 1) Please convert these shell script tests to use bash-like syntax, that is `[[` instead of `[` and `]]` instead of `]`.

**Comment by headcr4sh** [comment]
I really like the contents of the PR and would like to merge it as soon as possible.
I have just reviewed the sources of the PR and made some remarks.

File: assets/common.sh (hunk 1 above)
Comment (SimonXming) [reply assets/common.sh]: Sure 😄

**Comment by SimonXming** [comment]
> I really like the contents of the PR and would like to merge it as soon as possible.
> I have just reviewed the sources of the PR and made some remarks.

Thanks, I have run `shellcheck` and update the code according to the `shellcheck ` suggestion.

**Comment by SimonXming** [comment]
> I really like the contents of the PR and would like to merge it as soon as possible.
> I have just reviewed the sources of the PR and made some remarks.

Thanks, I have run `shellcheck` and update the code according to the `shellcheck ` suggestion.

**Pull request merged**